src/Land/Model/soil_heat.jl

```diff
@@ -1,35 +1,250 @@
 ### Soil heat model
+
 export SoilHeatModel, PrescribedTemperatureModel
 
 abstract type AbstractHeatModel <: AbstractSoilComponentModel end
 
 """
-    PrescribedTemperatureModel{FT} <: AbstractHeatModel
+    struct PrescribedTemperatureModel{F1} <: AbstractHeatModel
```

Comment:
```suggestion
    PrescribedTemperatureModel{F1} <: AbstractHeatModel
```
`struct` is a bit repetitive when looking at the doc string, since documenter [automatically tells you that it's a type](https://clima.github.io/ClimateMachine.jl/latest/APIs/Atmos/AtmosModel/#AtmosModel-balance-law)